plugins/JsTrackerInstallCheck/lang/en.json

```diff
@@ -0,0 +1,10 @@
+{
+    "JsTrackerInstallCheck": {
+        "PluginDescription": "Validate whether the tracking code has been installed on your site and that it can send tracking requests back to your Matomo instance",
+        "TestInstallationDescription": "To test that the tracking code is able to send requests to your Matomo instance, enter the exact URL you are tracking and click on \"Test installation\". A new tab might open, and should close within 10 seconds.",
```

Comment:
Actually, this was intentional. Javier specifically requested it say `might` because they might not even notice the tab open if their JS tracking code loads fast enough.